package.json

```diff
@@ -152,7 +152,7 @@
         "mime-types": "2.1.35",
         "modclean": "3.0.0-beta.1",
         "moment": "2.29.4",
-        "mongodb": "^2.2.36",
+        "mongodb": "~4.17.0",
```

Comment:
I wonder why the version is not exact, in compliance with [.npmrc](https://github.com/Inist-CNRS/lodex/blob/5ad6a25d6bd92e916320b223211e6a4ca82716b4/.npmrc#L2C18-L2C18)...